VideoApi/VideoApi.IntegrationTests/Api/AudioRecordings/GetAudioRecordingLinkTests.cs

```diff
@@ -16,6 +16,7 @@
 namespace VideoApi.IntegrationTests.Api.AudioRecordings;
 
 [Category("azurite")]
+[Obsolete("Those tests are failing locally but they will pass on the pipeline.")]
```

Comment:
They will also pass locally, you will need to run azurite. Checkout `docker-compose.tests.yml`

Reply:
attribute removed